hack/functest.sh

```diff
@@ -1,22 +1,55 @@
 #!/bin/bash
 
-set -e -o pipefail
+set -ex -o pipefail
 
-echo "Building e2e test suite (TODO)"
-#make build-e2e-test
+echo "Building e2e test suite"
+make build-e2e-test
 
-echo "Starting and preparing kubevirtci cluster"
+echo "Starting kubevirtci cluster"
 make cluster-up
 
 echo "Building and installing cloud-provider-kubevirt manager container"
 make cluster-sync
 
-echo "Running e2e test suite (TODO)"
-#export KUBECONFIG=$(./kubevirtci kubeconfig)
-#export NODE_VM_IMAGE_TEMPLATE=quay.io/capk/ubuntu-container-disk:20.04
-#export IMAGE_REPO=k8s.gcr.io
-#export TENANT_CLUSTER_KUBERNETES_VERSION=v1.21.0
-#export CRI_PATH=/var/run/containerd/containerd.sock
-#export ROOT_VOLUME_SIZE=23Gi
-#export STORAGE_CLASS_NAME=rook-ceph-block
-#make e2e-test
+export BIN_DIR=${BIN_DIR:-bin}
+export INFRA_KUBECONFIG=$(./kubevirtci kubeconfig)
+export KUBECONFIG=$(./kubevirtci kubeconfig)
```

Comment:
I do use infra kubeconfig as infra, and tenant as tenant.
I can't use kubeconfigs in `config/secret`, at least not directly, because those kubeconfigs have different api server address than I need when using it from the locahost.

Reply:
I can use `config/secret/kubeconfig` for the tenant client, but I'm not sure I can use the infra-kubeconfig, because in $(./kubevirtci kubeconfig), the sercer address is 
`server: https://127.0.0.1:49262`. Is the port always the same? It seems random to me.
Once we have the tests running in the infra cluster, we can use infra-kubeconfig directly